.github/workflows/verify-app.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Capture Growth Chart: verify app'
+
+on:
+  push:
+    branches:
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
```

Comment:
Using `node-version: ${{ matrix.node-version }}` allows us to easily modify or add support for different Node.js versions without the need to directly edit the configuration file. Please change the node version config in the lint job also.

Reply:
You can add a **strategy** at the bottom of the page: 
```
strategy:
    matrix:
      node-version: [18.x]
```